### DISCUSSION TITLE
How to export verifyingKey to wasm?

### DISCUSSION THREAD

**Discussion by readygo67** [Consensys/gnark#1117] [Ideas]
currently, we can use ExportSolidity to export verifyingkey to solidity,  can we support it to wasm? if yes, how?

**Comment by gbotrel** [comment]
gnark support for WASM is (very) limited at the moment. You can have a look at [this repo](https://github.com/vocdoni/gnark-prover-tinygo) which aims to adapt gnark to compile with TinyGo and get more reasonable perfs.

**Reply by ivokub** [reply]
Also, have a look at https://hackmd.io/@vocdoni/B1VPA99Z3 and https://github.com/vocdoni/gnark-prover-tinygo. This is for the prover part, but verifier part should me more straightforward using TinyGo.

**Comment by readygo67** [comment]
thanks, I will try them.
